src/hw4/index.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+// 2. Задача на обробники подій, роботу зі сховищами та атрибутами/вмістом
+// Напишіть html код який містить кнопку якраз посередині (вертикально і горизонтально) сторінки.
+// В початковому стані - на кнопці має бути текст 'Turn off', фон сторінки має стати світлий.
+// Після натиснення - на кнопці має бути текст 'Turn on', фон сторінки має стати темний.
+// Під кнопкою має з'явитись текстове повідомлення 'Last turn off: `{DD-MM-YYYY HH:MM:SS}`', де `{DD-MM-YYYY HH:MM:SS}` - це формат часу для виведення
+// Після повторного натиснення - на кнопці має бути текст 'Turn off', фон сторінки має стати світлий.
+// Під кнопкою має з'явитись текстове повідомлення 'Last turn on: `{DD-MM-YYYY HH:MM:SS}`', де `{DD-MM-YYYY HH:MM:SS}` - це формат часу для виведення
+// Стан кнопки та повідомлення останню зміну стану має зберігатись після перезавантаження/закриття сторінки.
+// Логіку роботи реалізуйте в окремому js-файлі.
+
+const button = document.querySelector(".button");
+const statusMessage = document.querySelector(".statusMessage");
+
+function turnOn() {
+  button.textContent = "Turn off";
+  document.body.classList.remove("dark-background");
+}
+
+function turnOff() {
+  button.textContent = "Turn on";
+  document.body.classList.add("dark-background");
+}
+
+function loadData() {
+  const isTurnedOn = localStorage.getItem("isTurnedOn");
+  const lastChange = localStorage.getItem("lastChange");
```

Comment:
```suggestion
  const isTurnedOn = localStorage.getItem("isTurnedOn");
  const lastChange = localStorage.getItem("lastChange");
```

краще ці значення отримувати на початку загрузки скрипта, а не під час виклику функцію в кінці

Reply:
На початку, це зразу за https://github.com/ChurychevaN/JS-HW/pull/4/files#diff-780420615ec9dc9d194803b13740b1ec2ed4952783ff4a285f820d82d28a3ddeR13

Це взагалі не має потреби виносити в окрему функцію